### PR TITLE
Akka 2.6.12

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

File: blog/_posts/2021-01-28-akka-2.6.12-released.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+title: Akka 2.6.12 Released
+author: Christopher Batey 
+short: We are pleased to announce the 2.6.12 patch release of Akka 2.6
+category: news
+tags: [releases]
+---
+
+Dear hakkers,
+
+We are pleased to announce a new patch release of Akka 2.6. This release is to fix a [memory leak in streams](https://github.com/akka/akka/issues/29966)
```
Comment (raboof) [inline blog/_posts/2021-01-28-akka-2.6.12-released.md:12]: Can we be more specific about what kinds of streams are affected, or is that not so easy to summarize in a few words?

**Review by raboof — APPROVED** [review]

File: blog/_posts/2021-01-28-akka-2.6.12-released.md (hunk 1 above)
Comment (johanandren) [reply blog/_posts/2021-01-28-akka-2.6.12-released.md:12]: "a memory leak affecting streams creating many nested streams with flat map concat or merge"?

Comment (patriknw) [inline blog/_posts/2021-01-28-akka-2.6.12-released.md:12]:
```suggestion
We are pleased to announce a new patch release of Akka 2.6. This release is to fix a [memory leak affecting streams creating many nested streams with flat map concat or merge](https://github.com/akka/akka/issues/29966)
```

**Comment by chbatey** [comment]
Thanks for finishing this off Patrik!

**Pull request merged**